_posts/2020-04-08-secure-software-development-lifecycle.md

```diff
@@ -0,0 +1,138 @@
+
+* Change management
+* Reading news (and vulnerability informations)
+* Incident and response
+* Monitoring and alerting
+* Bug bounty
+
+#### Retirement
+
+This is often forgotten in all projects. Some day the time of our beloved system will come to an end. At this point the system has had a great deal of organical growth and understanding the implications of the retirement can be hard to see. System needs to be taken down in a managed manner so that business can move to a new world, but at the same time all the resources of the system need to be disposed in a governed manner. Failing to due so will lead into an information leak. 
+
+Things that you should consider:
+* Migration plan
+* Archiving data (there might be laws)
+* Destroying virtual or physical resources
+* Cleaning up the data stores
+* Cleaning all the DNS names and such to avoid subdomain takeovers (especially with cloud)
+
+
+#### Example implementation
+
+People at Unity Tech have been so awesomely kind that they have published their SSDLC as [open source](https://github.com/UnityTech/unity-ssdlc/blob/master/Overview.md). I would strongly recommend looking at it as an example of an implementation.
+
+## My experience on the matter
+
+Taking security oriented tasks on the backlog has been natural since security has been brought up in front. Communication between different parties has been easy since we have had structure and documentation for the security conversations to take place. For example not a single threat modeling session has been in vain. It might feel before that we already know everything but there is always something we have missed.
+
+Key factor for making secure software development lifecycle to work is to find a spokesperson within your project for the matter. It does not need to be an extra guy but just somebody who likes to be a spokesperson for security matters. This person does not need know everything about information security as software development is always team work. It is just a good idea to have somebody who keeps on track about all security related solutions in your project and understands the greater picture.
+
+## Useful information sources
+
+The subject is not new and it has had many names. Microsoft SDL is maybe one of the first sources but it feels like more tailored towards product development. OWASP Testing Guide considers the software lifecycle with the same idea. NIST has had its own publications and there is also an ISO standard to dive in. Below is a list of useful sources that I have ventured on my journey. 
```

Comment:
considers the software lifecycle with the same idea <- not sure what this means?